core/field_angle.js

```diff
@@ -93,14 +93,36 @@ Blockly.FieldAngle.OFFSET = 0;
  */
 Blockly.FieldAngle.WRAP = 360;
 
-
 /**
  * Radius of protractor circle.  Slightly smaller than protractor size since
  * otherwise SVG crops off half the border at the edges.
  */
 Blockly.FieldAngle.RADIUS = Blockly.FieldAngle.HALF - 1;
 
 /**
+ * Adds degree symbol and recalculates width.
+ * Saves the computed width in a property.
+ * @private
+ */
+Blockly.FieldAngle.prototype.render_ = function() {
+  if (!this.visible_) {
+    this.size_.width = 0;
+    return;
+  }
+
+  // Update textElement.
+  this.textElement_.innerHTML = this.getDisplayText_();
```

Comment:
we usually try to avoid using innerHtml since it is a pretty heavy hammer that can lead to text escaping problems that we don't want to deal with.  Is there something else you can use here to update the value?